server/src/main/java/org/elasticsearch/search/builder/SearchSourceBuilder.java

```diff
@@ -1104,6 +1135,8 @@ public void parseXContent(XContentParser parser, boolean checkTrailingTokens) th
                     sliceBuilder = SliceBuilder.fromXContent(parser);
                 } else if (COLLAPSE.match(currentFieldName, parser.getDeprecationHandler())) {
                     collapse = CollapseBuilder.fromXContent(parser);
+                } else if (CreateIndexRequest.MAPPINGS.match(currentFieldName, parser.getDeprecationHandler())) {
```

Comment:
I would rename the section to runtime_mappings, at least for now. I think it better defines that it only holds mappings for runtime fields

Reply:
I guess. It is in the search request. But if we're already going to drop the need for `properties` then trying to line it up with the index creation api isn't really right anyway.